bin/jobs_functions.js

```diff
@@ -114,6 +122,36 @@ function terminateTestJob(testjobId) {
         })
 }
 
+function deleteTestJob(testjobId) {
+    let url = config.server + '/api/v1/testjobs'+ '/' + testjobId + '/delete';
```

Comment:
The slash can be part of the first string and doesnt need to be concateneted seperately.

'/api/v1/testjobs'+ '/' + testjobId + '/delete';
-> 
'/api/v1/testjobs/' + testjobId + '/delete'